src/bn.iced

```diff
@@ -1,6 +1,6 @@
 
 {nbv,nbi,BigInteger,nbits} = require 'bn'
-{buffer_to_ui8a} = require './util'
+{buffer_to_ui8a} = require('pgp-utils').util
```

Comment:
i don't think it's used in this file, or am I missing something?

Reply:
oh you're right - I spent the time fixing the require to not be circular...but didn't realize it isn't needed. I'll push a fix.